scripts/launch-xrd

```diff
@@ -397,9 +397,12 @@ else
             # Required for XRd vRouter to use vfio driver.
             "--device" "/dev/vfio"
         )
-        # Podman bug - it doesn't handle the first wildcard asterisk required
-        # in --device-cgroup-rule. However, Podman doesn't seem to restrict
-        # access to devices by default can just skip this arg for Podman.
+        # Podman bug - Podman's parsing of the first asterisk was only only
```

Comment:
Podman's parsing of the first asterisk ... in what? Need to mention '--device-cgroup-rule' earlier in the comment.